kura/org.eclipse.kura.linux.debian.provider/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,21 @@
+Manifest-Version: 1.0
```

Comment:
The bundle name should probably be `org.eclipse.kura.linux.net.debian.provider`. It should work with all Debian derivatives that use `ifupdown` to configure the networking